### ISSUE TITLE
Orientation of "turn on red" icon

### ISSUE THREAD

**Issue by aubergine10** [krzychu124/Cities-Skylines-Traffic-Manager-President-Edition#56]
The icon for the new **turn on red** feature, being an arrow, is directional. However it rotates to face camera position like all the other icons, thus from many angles it's pointing in a confusing direction.

Would it be possible to prevent this specific icon from rotating, so that it always points from the source road to the destination road, no matter what angle the camera is at?

It can tilt so that it's 'face' is pointing towards the camera, but it's rotation should be locked such that the pointy end of the arrow faces towards the destination road, and the other end of the arrow 'points' to the source road.

**Comment by aubergine10** [comment]
In image below, there are two places where **turn on red** is applicable.

* The indicated arrow icon is pointing the wrong way, it needs rotating 90 degrees
* the other arrow (bottom road) is facing right way, but only because of the current camera position

![turn on red](https://user-images.githubusercontent.com/1386719/52537076-53ef2980-2d5a-11e9-85e4-f7ec1cbecf78.gif)

**Comment by krzychu124** [comment]
I know, i know 😄 and I have no idea if it's possible since it's overlay relative to camera position. I'm currently digging in code how to render basic shapes 😅

**Comment by aubergine10** [comment]
Are there plans to have **turn on red** available for normal traffic lights? If not, and it's only going to be for **timed traffic lights**, then it might be worth moving it in to timed traffic lights UI at some point? Seems odd being a junction restriction when it's limited to just timed traffic lights...?

**Comment by aubergine10** [comment]
BTW an alternate to dynamic shape drawing would be to have 4 or 8 versions of the icon, where the image files themselves are rotated, and just switch between image files for the on-screen overlay as the camera moves? Grim, but might work as a quick fix.

**Comment by krzychu124** [comment]
__Turn on red__ is available for normal traffic lights 😉 

Good idea with shapes but i think it would be better to find some way to lock position to direction of road segment

**Comment by aubergine10** [comment]
hrm, I didn't notice the arrows for normal lights, testing...

**Comment by aubergine10** [comment]
Doh! Does help if the junction has traffic lights! Yes, it works on normal lights too! Awesome!

**Comment by FireController1847** [comment]
Just to note on this, this is confusing as well when setting manual traffic lights. The left arrow is on the left side, which makes sense, but if you turn the camera it appears to move to the right side when really it's still on the left, if you know what I mean.

**Comment by aubergine10** [comment]
IMO, "manual traffic lights" is a feature not worth spending time on (see #22).

**Comment by FireController1847** [comment]
My apologizes, I meant timed traffic lights.

**Comment by aubergine10** [comment]
IMO "timed traffic lights" UI needs a complete overhaul to make it usable by humans :)

**Comment by krzychu124** [comment]
Manual could be our sandbox to test features

**Comment by aubergine10** [comment]
True, we could use it to beta test #8 (timed traffic lights overhaul)  :)
